lib/math.js

```diff
@@ -26,13 +26,13 @@ class CryptoRandomPrefetcher {
 const cryptoPrefetcher = (
   // Create prefetcher to use when crypto.randomBytes is required to generate
   // multiple same-size values. `bufSize` must be a multiple of `valueSize` for
-  // this to work.
+  // this to work
```

Comment:
Why did you remove a period here and leave it on line 53 below?
Also, I think it makes sense to have a period at the end of all full sentences when a comment has more than one sentence in it.